src/themes/mainTheme.ts

```diff
@@ -353,6 +353,22 @@ export const mainTheme = createTheme(
           },
         },
       },
+      MuiPickersFilledInput: {
```

Comment:
Må fikse styling av filled DatePicker field som ikke vikrer per nå 

![bilde](https://github.com/user-attachments/assets/be7c102a-307d-47e5-b4d8-9e8da0f903eb)

Dette er beskrevet under "Migrate the theme" her: https://mui.com/x/migration/migration-pickers-v7/

Ser ut som dette skal virke, ref https://github.com/mui/mui-x/blob/d7bd7cb84ffcc5c6a20cf87a102b6ba680567139/packages/x-date-pickers/src/themeAugmentation/themeAugmentation.spec.ts#L663

Reply:
Fant ut av denne. Må bare importere pickers typen: https://github.com/BIBSYSDEV/NVA-Frontend/pull/7328/files#diff-1b4b25b3be213d79ca0c43097b5c13119cacdca261f4f13a8d35f18d76b3579c